data/avgi/avgi side missions.txt

```diff
@@ -0,0 +1,1070 @@
+				`	"Do you just test nukes here?"`
+					goto fun
+
+			`	"It's nice to have a big mass to brace experiments against and bury sensors in, explanation. Plus we get to see the effects of weapons on bulk mass, like all this ice and the occasional rock outcropping. It makes quite the blast plume," she adds happily.`
+			`	"We used to use some big asteroids for this, but we kept blowing through them, amusement. Relocating our lab after every big test was getting annoying, so we picked this planetoid out and stuck an outpost on it."`
+			choice
+				`	"Why this planet in particular?"`
+				`	"Blowing through asteroids? Just how big are these nukes?"`
+					goto big
+
+			`	"Consideration." She looks thoughtful. "Well, the lab managers told us to go somewhere far enough from the Navigeo Yards to be out of the way of anything important, so that they don't have to worry about getting flashed with radiation every week. But really I think it's because they wanted their own little planetoid as a personal fief where no one could tell them what to do, and Dokimi is so utterly unremarkable that no one else wanted it for anything else."`
+				goto next
+
+			label big
+			`	"Heh, we can make them as big as we want to. 'Just add more lithium deuteride,' as they say, amusement. Once you get a proper two-stage device, you can pretty much just add more stages to boost the yield. Two or three hundred megatons is easy, if a bit impractical to fit in the sizes of missile we field. But we sometimes test at that range, to validate other ideas and designs."`
+				goto next
+
+			label fun
+			`	"Nah, we run all sorts of fun stuff that we can't test on Ergastirio, appreciation. The really big particle beams, for example, and reactor pumped lasers that would be annoying to shield on the station."`
+			`	She gestures out the window, towards some stony mountains. "You can find a valley out that way with all sorts of tracks carved in the ice. I wrote my name in a patch of stone, actually.`
+				goto next
+
+			label next
+			`	At Aliza's direction, you set the <ship> down on an icy pad by one of the small buildings. A few Avgi come out to greet you, wearing silver-colored spacesuits with comically large bubble helmets.`
+			choice
+				`	"What now?"`
+				`	"How long until the test?"`
+			`	Aliza checks a clock. "We've got most of the day. Want to help us set up, question?"`
+			`	"Sure, I've got nothing better to do," you say.`
+			`	What follows is a surprisingly mundane day of helping the crew set up various sensors and instruments at carefully marked distances from ground zero. As you're helping drill a borehole through the ice, through which yet another probe will be dropped, you see a sled in the distance drive by, an innocent-looking teardrop object strapped to its bed. It can't be any bigger than you yourself are, yet this must be the device that will be tested in only a few hours, raised on top of a slim steel tower that has already been erected.`
```

Comment:
```suggestion
			`	What follows is a surprisingly mundane day of helping the crew set up various sensors and instruments at carefully marked distances from ground zero. As you're helping drill a borehole through the ice, through which yet another probe will be dropped, you see a sled in the distance drive by, an innocent-looking teardrop-shaped object strapped to its bed. It can't be any bigger than you yourself are, yet this must be the device that will be tested in only a few hours, raised on top of a slim steel tower that has already been erected.`
```